### PR TITLE
Replace println with log (#921)

### LINKED ISSUE #342: Use logging instead of raw printing for prover workspace member reports
Currently, we are just printing the info of the CLI bin with `println!`. 

But @ed255 introduced some time ago logging into the workspace. So it would be nice to use it also in the `prover` workspace member.

This can be taken as implementation example: https://github.com/appliedzkp/zkevm-circuits/blob/28b1ea7a6d3e70fa7b20f47fe24b850c7a48d8b3/integration-tests/src/bin/gen_blockchain_data.rs#L87

### PR DESCRIPTION
**Pull request by SuccinctPaul**
see more on #342

### CODE REVIEW

**Comment by ed255** [comment]
@ChengYueJia it seems that clippy detected some issues, could you fix them?  You can test clippy localy by running `make clippy` from the root of the repository.

**Comment by ChengYueJia** [comment]
> @ChengYueJia it seems that clippy detected some issues, could you fix them? You can test clippy localy by running `make clippy` from the root of the repository.

Thanks for your help. It's done.

File: zkevm-circuits/src/state_circuit.rs [hunk 1]
```diff
@@ -593,13 +594,13 @@ mod state_circuit_stats {
...
-        println!(
+        info!(
             "| {: <14} | {: <14} | {: <2} | {: >6} | {: <5} |",
             "state", "opcode", "h", "g", "h/g"
         );
-        println!("| ---            | ---            | ---|    --- | ---   |");
+        info!("| ---            | ---            | ---|    --- | ---   |");
         for (state, opcode, height, gas_cost) in stats {
-            println!(
+            info!(
```
[the remark this answers is not in the document]
Comment (CPerezz) [reply zkevm-circuits/src/state_circuit.rs:603]: ping @ChengYueJia

**Review by CPerezz — COMMENTED** [review]
If the tables are displayed correctly, I'll approve and merge. Could you test it locally @ChengYueJia ??

**Comment by CPerezz** [comment]
@ChengYueJia I suggest to resolve the conflicts, check the table correctness and merge this. As the more time it passes, the more issues this PR will encounter.

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,4 @@
 *.png
 .DS_Store
 .vscode
+.idea
```
[the remark this answers is not in the document]
Comment (ChengYueJia) [reply .gitignore:5]: this is generated by the Intellj IDEA(And Editor).

File: keccak256/src/arith_helpers.rs [hunk 3]
```diff
@@ -192,7 +193,7 @@ pub fn inspect(x: BigUint, name: &str, base: u8) {
     let mut chunks = x.to_radix_le(base.into());
     chunks.resize(65, 0);
     let info: Vec<(usize, u8)> = (0..65).zip(chunks.iter().copied()).collect_vec();
-    println!("inspect {} {} info {:?}", name, x, info);
+    log!("inspect {} {} info {:?}", name, x, info);
```
[the remark this answers is not in the document]
Comment (ChengYueJia) [reply keccak256/src/arith_helpers.rs]: I'll modify it.

**Comment by ChengYueJia** [comment]
> @ChengYueJia I suggest to resolve the conflicts, check the table correctness and merge this. As the more time it passes, the more issues this PR will encounter.

Sorry for response for it so later. I'll resolve all of them soon. I'll leave common above..

File: zkevm-circuits/src/state_circuit.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (ChengYueJia) [reply zkevm-circuits/src/state_circuit.rs:603]: This is a problem.
Actually, seems like there is no need to replace the println with log in test part. 
The debug is for showing the zkevm runtime info. 
The println can be used for the unit test, which runs locally.
Am I right?

I'll rollback the modify above.

**Comment by ChengYueJia** [comment]
And I've modify it.
please have another view.

File: zkevm-circuits/src/state_circuit.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (CPerezz) [reply zkevm-circuits/src/state_circuit.rs]: Agree! I missed that this was the test module already.

Please, revert this.

**Review by CPerezz — APPROVED** [review]
Lgtm! Thanks for this work!

**Comment by CPerezz** [comment]
@ChengYueJia I'd need you to resolve the conflicts so that we can merge

**Comment by ChengYueJia** [comment]
@CPerezz sorry, I have submitted a new pr. #963

**Pull request closed without merging**